task/control-plane-agent/src/mgs_sidecar.rs

```diff
@@ -314,6 +315,21 @@ impl SpHandler for MgsHandler {
         }
     }
 
+    fn component_action(
+        &mut self,
+        _sender: SocketAddrV6,
+        component: SpComponent,
+        action: ComponentAction,
+    ) -> Result<(), SpError> {
+        match (component, action) {
+            (SpComponent::SYSTEM_LED, ComponentAction::Led(action)) => {
+                // TODO: implement this
```

Comment:
Do you want to do this before merging, or does it require other work first?

Reply:
This is pending @Aaron-Hartwig 's work, because the system LED is actually controlled by one of the front IO FPGAs.

Reply:
I agree this is fine to leave as a TODO, when Ben and I get the main transceiver management rewrite through I'm hopping onto the LEDs.